### PR TITLE
Add completion for rmlint

### PR DESCRIPTION
**Pull request by oxiedi**
- [x] This compdef is not already available in zsh.
- [x] This compdef is not already available in their original project.
- [x] I am the original author, or I have authorization to submit this work.
- [ ] This is a finished work.
- [x] It has a header containing authors, status and origin of the script.
- [x] It has a license header or I accept that it will be licensed under the terms of the Zsh license.

### CODE REVIEW

File: src/_rmlint.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#compdef rmlint.sh -P rmlint.*.sh
```
Comment (syohex) [inline src/_rmlint.sh:1]: Is this file necessary ? I installed `rmlint` but there is no `rmlint.sh` file.

Comment (oxiedi) [reply src/_rmlint.sh:1]: This is produced by `rmlint` itself:

    % rmlint >/dev/null
    % ./rmlint.sh -h
    usage: ./rmlint.sh OPTIONS

    OPTIONS:

      -h   Show this message.
      -d   Do not ask before running.
      -x   Keep rmlint.sh; do not autodelete it.
      -p   Recheck that files are still identical before removing duplicates.
      -r   Allow deduplication of files on read-only btrfs snapshots. (requires sudo)
      -n   Do not perform any modifications, just print what would be done. (implies -d and -x)
      -c   Clean up empty directories while deleting duplicates.
      -q   Do not show progress.
      -k   Keep the timestamp of directories when removing duplicates.
      -i   Ask before deleting each file
    % rmlint --backup >/dev/null
    % ./rmlint.2021-07-16T12:00:00Z.sh -h
    ...

Comment (syohex) [reply src/_rmlint.sh:1]: Thanks for information. I'll check it later

**Comment by syohex** [comment]
@oxiedi Your completion files looks good to me. Could you add license header to completion files ?

**Comment by oxiedi** [comment]
> Could you add license header to completion files ?

Doesn't lack of the license header mean I'm OK with the Zsh license (I am, BTW)? Should I include the Zsh license in the header then?

**Comment by syohex** [comment]
> Should I include the Zsh license in the header then?

Yes, please.

**Comment by oxiedi** [comment]
Done.

> It has a license header or I accept that it will be licensed under the terms of the Zsh license.

This point is misleading then.

**Comment by syohex** [comment]
Thanks for your work.

> This point is misleading then.

Sorry. I'll try to update pull request template later.
